_data/locales/hu-HU.yml

```diff
@@ -79,7 +79,7 @@ post:
 categories:
   category_measure:
     singular: kategória
-    plural: kategóriák
+    plural: kategória
   post_measure:
     singular: bejegyzés
-    plural: bejegyzések
+    plural: bejegyzés
```

Comment:
If there is no need to distinguish between singular and plural forms, it can be modified as follows:

```yaml
categories:
  category_measure: kategória
  post_measure: bejegyzés
```

Reply:
Thanks! That made it better.